scvi/external/cell2location/tutorial/cell2location_analysis_2_estimating_signatures.ipynb

```diff
@@ -0,0 +1,1466 @@
+{
```

Comment:
<p>This is alright for now, but you want people to be able to fully run the analysis and not rely on external software</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tools/pull/930/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>